### ISSUE TITLE
Priority of the WAN interface over the VPN

### ISSUE THREAD

**Issue by sim-13** [SWRT-dev/rtax89x#1]
There is an error in the original ASUS firmware that they do not
they haven't been fixed for several years: the priority of the WAN interface over VPN tunnels, i.e. if you create a VPN connection to a server and set certain routes to it, then when clients request these routes, they still go through the WAN interface, and not through the VPN tunnel. In the MERLIN firmware, this has been fixed for a long time. But in this firmware, RTAX89X_R5.1.5b1-gce1a50e. trx the problem is still present. Can you fix it?

**Comment by paldier** [comment]
No, the vpn part of asus is closed code, i dont't want to rewrite it. But i will forward your question to asus.
If you can port the vpn part of merlin then you can submit a pull request.

**Comment by sim-13** [comment]
I see. I contacted ASUS with AX88U and AX89X about this - they create a ticket and forget about it... But in the case of AX88U-there is firmware from MERLIN and everything works correctly there, but AX89X does not...

**Comment by sim-13** [comment]
> No, the vpn part of asus is closed code, i dont't want to rewrite it. But i will forward your question to asus.
> If you can port the vpn part of merlin then you can submit a pull request.

Hi. Did asus answer something?

**Comment by paldier** [comment]
Nothing, we can only port by ourselves.

**Comment by sim-13** [comment]
Well, as expected... Unfortunately, I can't do this myself. I can only test it on my own equipment.
